src/case.c

```diff
@@ -13,42 +13,45 @@
 #include "case.h"
 
 #define CASE_MODIFIER     0x20
```

Comment:
It looks like this is no longer used. Should we remove it? I can do this in a separate commit if you don't want to.